### PR TITLE
Upgrade the containers to be based on Debian Buster

### PR DESCRIPTION
**Pull request by saiarcot895**
Signed-off-by: Saikrishna Arcot <sarcot@microsoft.com>

### CODE REVIEW

**Comment by sumukhatv** [comment]
Why are we changing the Makefile?

**Comment by saiarcot895** [comment]
That's from e06e8c9, to enable position-independent executables to be created and to have stack-smashing protection compiled in.

**Comment by sumukhatv** [comment]
> That's from [e06e8c9](https://github.com/Azure/sonic-restapi/commit/e06e8c9e06b0a08c1014cde497dea13ada4b9fd6), to enable position-independent executables to be created and to have stack-smashing protection compiled in.

I would suggest making them two separate PRs. Changing the base image in itself is a major change. Let's keep this PR exclusively for that so we can revert back if it causes any issue in the final image we build.

**Comment by saiarcot895** [comment]
Done. I kept the `make clean` change in.

**Review by sumukhatv — APPROVED** [review]

File: azurepipeline.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ jobs:
     - script: |
         set -ex
         ./build.sh
+        docker ps
```
[the remark this answers is not in the document]
Comment (saiarcot895) [reply azurepipeline.yml]: This was a temporary debugging change.

File: Makefile [hunk 2]
```diff
@@ -27,7 +27,9 @@ $(GOPATH)/src/go-server-server/main.go:
 	cp -r swsscommon       $(GOPATH)/src/swsscommon
 
 libcswsscommon:
-	cd libcswsscommon && sudo make install
+	make -C libcswsscommon
```
[the remark this answers is not in the document]
Comment (saiarcot895) [reply Makefile:30]: `-C` tells `make` to change to the specified directory before doing anything.

**Pull request merged**